### ISSUE TITLE
Problem for repack to EngText .subp (the game is shutting down)[MGSV:TPP]

### ISSUE THREAD

**Issue by SLaYeR983** [Atvaark/FoxEngine.TranslationTool#18]
I using TranslationTool v0.2.5 or v0.2.6. the files (EngText) unpack, modding and repack working perfectly on Japanese or Portuguese (utf-8). But when unpack and repack (ISO8859-1) to EngText .subp files, the game freezes and shutting down, especially when I load the game from the checkpoint.

I using to compatible word characters with English(8859-1). (https://kb.iu.edu/d/aepu). Is there limit file size or other any reason...? the modded file size must equal with original .subp or .subp.xml, whatever the reason? Can't pack it in EngText (English)? Can you help me please... One choice is pack it other languages? 

I want to this because of do not understanding in other languages' online FOB and other missions texts.

**Note:** My subp files sizes are smaller than the originals, all of them.

**Comment by abuali129** [comment]
I remember that happened to me too... That's is why i switched to mod Japanese language.
May be you tried to change the glyphs to show another character than what is written, then may be those characters breaks the xml file and causing crashes.
Just pick another language, take the texts from English files to the corresponding Japanese ones then repack, you will have full English texts on Japanese option

**Comment by SLaYeR983** [comment]
Yes, I changed the appearance of the letters, but I used only compatible ISO8859-1 characters. and still take crash...

Do I have a chance to adaptation, online english texts to other languages? Very annoying to see incomprehensible text in the interface and when starting the game.

**Comment by abuali129** [comment]
You mean server messages? I don't think so.
The latin extended letters causes crashes when written for English xml to show on English option for the game.

**Comment by SLaYeR983** [comment]
Atvaark did great work, but the tool may be doing something wrong?

**Comment by abuali129** [comment]
No, it is the game, when ever you write any latin extended character it will crash

**Comment by SLaYeR983** [comment]
OK... but there is not latin extended in the modded files. That all basic latin and latin-1 characters

first of all, I changed only "Ş ş İ ı Ğ ğ" to "þ ø ð" and crashed again. the problem is not only latin extended, I suspecting the file size limit.

**Comment by abuali129** [comment]
Will I'm not expert with the title of these characters, I mean those Latin characters with punctuations that you mentioned...

**Comment by SLaYeR983** [comment]
I understand. I said tried partially, added just þ ø ð to strings, the result is same. These characters are compatible with ISO8859-1. I guess Atvaark finds a solution.

for example: "Yá’át’ééh, Bilagáana, Nizhóní" these are in the original texts, but I don't know why take crash when added others. maybe using special character encoding in the game for english.

**Comment by SLaYeR983** [comment]
Wouldn't you help with that Atvaark? :((
